deploy/klusterlet/chart/klusterlet/templates/cluster_role.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if or ( eq .Values.klusterlet.mode "Singleton") (eq .Values.klusterlet.mode "Default") ( eq .Values.klusterlet.mode "") }}
```

Comment:
can mode be an empty string

Reply:
checked that there is a default value for mode in values.yaml. 
removed the empty check here.